docs/UserGuide.md

```diff
@@ -219,6 +219,21 @@ Examples:
 * `add_event n/Meeting d/CS2103T Meeting f/2024-09-09 t/2024-09-10`
 * `add_event n/Workshop d/Orbital Workshop f/2024-10-01 t/2024-10-10`
 
+### Editing an event: edit_event
+
+Edits the details of an existing event in the address book.
+
+Format: `edit_event INDEX n/EVENT_NAME d/EVENT_DESCRIPTION f/EVENT_START_DATE t/EVENT_END_DATE`
+
+* The INDEX is required to identify which event to edit.
+* The date inputs must be in the format YYYY-MM-DD.
+* At least one of the optional fields (EVENT_NAME, EVENT_DESCRIPTION, EVENT_START_DATE, EVENT_END_DATE) must be provided to make changes.
```

Comment:
Please surround EVENT_NAME, EVENT_DESCRIPTION, EVENT_START_DATE, EVENT_END_DATE with ` on each of them to be coherent with the rest of the user guide.